src/main/java/com/UnitOmega13/entity/User.java

```diff
@@ -1,4 +1,4 @@
-package com.UnitOmega13.demo.entity;
+package com.UnitOmega13.entity;
```

Comment:
@Column(name = "userId") -> @Column(name = "user_id")
@Column(name = "accessRole") -> @Column(name = "access_role")
do we need constructors with parameters?